### PR TITLE
Add Danvers Circuit City and Lowell General Hospital sites as a new mass vaccination sites

### PR DESCRIPTION
**Pull request by madeleine-b**
Per this [article](https://dartmouth.theweektoday.com/article/dartmouth-circuit-city-be-mass-vaccine-site/51758), there will be a new mass vaccination site in Dartmouth, MA beginning February 24. This change adds the site to the list of "mass vaccination sites" that appear with a red star when there is availability. See lower right part of state

Screen Shot 2021-02-15 at 3 01 16 PM

Also, fixes a bug in which some sites appeared with a colorful star even if they didn't have availability. This was caused by the order in which we checked the details of a location vs checking if it had appointments available.

### CODE REVIEW

File: views/index.ejs [hunk 1]
```diff
@@ -107,27 +107,27 @@
...
               scale = 5
+            } else if (element.fields['Serves']?.trim() === "Eligible populations statewide") {
+              icon_logo = "https://upload.wikimedia.org/wikipedia/commons/thumb/1/1f/Green_star_41-108-41.svg/807px-Green_star_41-108-41.svg.png"
+              scale = 5
+            } else {
+              icon_logo = "<%= static_path + "/200px-Blue_Star.png" %>"
+              scale = 5
+            }
+
+            if (element.fields['Availability']?.trim() === 'None') {
```
Comment (johndeyrup) [inline views/index.ejs:128]: can just use loc here

File: views/index.ejs [hunk 2]
```diff
@@ -107,27 +107,27 @@
...
-              icon_logo = "<%= static_path + "/200px-Blue_Star.png" %>"
-              scale = 5
-            }
             const loc = element.fields['Location Name']?.trim();
             if (
               loc === "Foxborough: Gillette Stadium"
               || loc === "Boston: Fenway Park"
               || loc === "Danvers: Doubletree Hotel"
               || loc === "Springfield: Eastfield Mall"
+              || loc === "Dartmouth: Circuit City at Dartmouth Towne Center"
```
Comment (mzagaja) [inline views/index.ejs]: What do you think about this refactor?

```javascript
const massVaccinationSites = [
"Foxborough: Gillette Stadium",
"Boston: Fenway Park",
"Danvers: Doubletree Hotel",
"Springfield: Eastfield Mall",
"Dartmouth: Circuit City at Dartmouth Towne Center"
]

if(massVaccinationSites.includes(loc)) { .. }
```

File: views/index.ejs [hunk 3]
```diff
@@ -107,27 +107,27 @@
...
             ) {
               icon_logo = "https://upload.wikimedia.org/wikipedia/commons/3/34/Red_star.svg"
               scale = 5
+            } else if (element.fields['Serves']?.trim() === "Eligible populations statewide") {
+              icon_logo = "https://upload.wikimedia.org/wikipedia/commons/thumb/1/1f/Green_star_41-108-41.svg/807px-Green_star_41-108-41.svg.png"
```
Comment (mzagaja) [inline views/index.ejs]: This might be worth copying into our local images folder and then loading it locally to reduce cross network requests.

**Comment by mzagaja** [comment]
From a functional standpoint this is good to merge! I had a couple suggestions you can add to this PR if you would like.

**Review by mzagaja — APPROVED** [review]

File: views/index.ejs (hunk 1 above)
Comment (madeleine-b) [reply views/index.ejs:128]: Ah actually wait, |loc| refers to the location name, so I'm not sure where you were suggesting I use it.

**Comment by madeleine-b** [comment]
Thanks @mzagaja! In the time since I opened the PR, I found out about another mass vax site in Lowell and added that. Made the refactoring changes too.

**Pull request merged**